lib/jq.ex

```diff
@@ -92,9 +92,13 @@ defmodule JQ do
           raise(SystemCmdException, result: error, command: "jq", args: [query, file_path])
 
         {value, code} when is_integer(code) and code == 0 ->
-          result = Poison.decode!(value)
-          unless result, do: raise(NoResultException)
-          result
+          case Poison.decode!(value) do
```

Comment:
everything else was dealing with enough deprecations and warnings to compile. This is the meat of the change.